### DISCUSSION TITLE
[Q] How to Copy Text Without Citation in Better Note?

### DISCUSSION THREAD

**Discussion by doosik71** [windingwind/zotero-better-notes#1232] [Q&A] [answered]
Here’s the translation into English:

---
When copying the word 'Abstract' from a PDF document and pasting it into Better Note, it gets entered as ‘“Abstract” (Chen et al., 2022, p. 1)’.  
Is there a way to toggle the function so that only the text is copied, without the citation?

**Comment by windingwind** [comment, marked as answer]
Ctrl+shift+v to paste as plain text
